microprofile.ls/org.eclipse.lsp4mp.ls/src/main/java/org/eclipse/lsp4mp/commons/MicroProfileJavaHoverParams.java

```diff
@@ -26,16 +26,18 @@
 	private String uri;
```

Comment:
Please copy/paste MicroProfileJavaHoverParams.java from JDT side. Those class must be the same.